src/ExcelDataReader/Core/BinaryFormat/XlsWorkbook.cs

```diff
@@ -163,8 +163,6 @@ internal void AddXf(XlsBiffXF xf)
         private void ReadWorkbookGlobals(XlsBiffStream biffStream)
         {
             XlsBiffRecord rec;
-            var biffFormats = new Dictionary<ushort, XlsBiffFormatString>();
```

Comment:
FYI, the purpose of the temporary biffFormats dict was to make sure the CODEPAGE record is parsed before decoding the format strings. The new code assumes the CODEPAGE record comes before the FORMAT records. It's probably OK as-is by the spec, although I'd be happier if we could revert these changes in XlsWorkbook.cs before merging